org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/elementmodel/LanguageUtils.java

```diff
@@ -935,11 +964,23 @@ private XhtmlNode adjustToLang(XhtmlNode xhtml, String lang, String status, Stri
   public boolean switchLanguage(Element e, String lang, boolean markLanguage, String resourceLang, String defaultLang, List<ValidationMessage> errors) throws IOException {
     boolean changed = false;
     if (e.getProperty().isTranslatable()) {
-      String cnt = getTranslation(e, lang);
-      e.removeExtension(ExtensionDefinitions.EXT_TRANSLATION);
-      if (cnt != null) {
-        e.setValue(cnt);
-        changed = true;
+      if (keepTranslationsWhenTranslating) {
```

Comment:
Same here? Translate, but modify an extension so that it can reverse the translation?